client/synapseJavaClient/src/main/java/org/sagebionetworks/client/SynapseClient.java

```diff
@@ -1499,11 +1498,15 @@ public String downloadCsvFromTableAsyncStart(String sql, boolean writeHeader, bo
 	/**
 	 * Return a list of Teams given a list of Team IDs.
 	 * 
+	 * Note: Invalid IDs in the list are ignored:  The results list is simply
+	 * smaller than the set of IDs passed in.
+	 *
+	 * 
 	 * @param ids
 	 * @return
 	 * @throws SynapseException
 	 */
-	public ListWrapper<Team> listTeams(IdList ids) throws SynapseException;
+	public List<Team> listTeams(IdSet ids) throws SynapseException;

```

Comment:
Rather see these as standard Set<Long> parameters that we convert into an IdSet for the service call.